### DISCUSSION TITLE
glTF查看器代码有开源吗

### DISCUSSION THREAD

**Discussion by hcxmj** [galacean/discussion#84] [Editor] [answered]
想要看下glTF查看器初始化时相机&控制器角度是怎么设置的，找了一下代码仓库里好像没有看见glTF网页查看代码
<img width="1411" alt="image" src="https://github.com/galacean/discussion/assets/58494807/cae64ceb-0487-47e8-b08d-605fe0df723f">

**Comment by eyworldwide** [comment, marked as answer]
@hcxmj  你好，源码在这里：https://github.com/galacean/galacean.github.io/tree/main/src/components/GltfViewer

**Reply by hcxmj** [reply]
好的，谢谢啦！
